apps/taxes/src/modules/avatax/avatax-error-normalizer.ts

```diff
@@ -0,0 +1,17 @@
+import { createLogger } from "../../lib/logger";
+import { AvalaraError } from "avatax/lib/AvaTaxClient";
+import { TaxCriticalError, TaxUnexpectedError } from "../taxes/tax-error";
+
+export class AvataxErrorNormalizer {
+  private logger = createLogger({ name: "AvataxErrorNormalizer" });
+
+  normalize(error: unknown) {
+    if (error instanceof AvalaraError) {
+      this.logger.debug(error.stack, "AvalaraError occurred");
```

Comment:
Why is it `debug` level and not `error`?

Reply:
I wanted to log the original error as a debug. The processed error is logged as `.error` in `webhook-response.ts`.